numpyro/distributions/util.py

```diff
@@ -414,6 +414,37 @@ def clamp_probs(probs):
     return jnp.clip(probs, a_min=finfo.tiny, a_max=1.0 - finfo.eps)
 
 
+def betainc(a, b, x):
+    try:
+        from tensorflow_probability.substrates.jax.math import betainc as betainc_fn
+    except ImportError:
```

Comment:
do you want to include an error message here as below?

Reply:
Here we use the default jax implementation (which does not have grad over all params)